README.md

```diff
@@ -1,5 +1,5 @@
 # openIMIS Frontend Template module
-This repository holds the files of the openIMIS Frontend Template module.
+This repository holds the files of the openIMIS Payment Cycle module.
 It is dedicated to be bootstrap development of [openimis-fe_js](https://github.com/openimis/openimis-fe_js) modules, providing an empty (yet deployable) module.
```

Comment:
Please add more information to documentation (like in another frontend modules)

Reply:
done